lib/complex.ex

```diff
@@ -893,38 +893,41 @@ defmodule Complex do
 
   @doc """
   Returns a new complex that is the complex natural log of the provided
-  complex number, $ln(z) = log_e(z)$.
+  complex number, $log(z) = log_e(z)$.
```

Comment:
Should I make this

```suggestion
  complex number, $log(z) = ln(z)$.
```

?

Reply:
I think it should be ln(z) = log_e(z)

Reply:
Just so we're more explicit about what's happening